### ISSUE TITLE
DNS export records to a file

### ISSUE THREAD

**Issue by blowmage** [GoogleCloudPlatform/gcloud-ruby#301]

**Comment by quartzmo** [comment]
I'm debating whether or not to sort the Records before exporting. Most importantly, this sort would put the SOA record at the top. The gcloud command-line tool's [dns record-sets export](https://cloud.google.com/sdk/gcloud/reference/dns/record-sets/export) command does *not* appear to perform a sort. Here is its output:

```
example.net. 21600 IN NS ns-cloud-b1.googledomains.com.
example.net. 21600 IN NS ns-cloud-b2.googledomains.com.
example.net. 21600 IN NS ns-cloud-b3.googledomains.com.
example.net. 21600 IN NS ns-cloud-b4.googledomains.com.
example.net. 21600 IN SOA ns-cloud-b1.googledomains.com. dns-admin.google.com. 0 21600 3600 1209600 300
www.example.net. 18600 IN A 127.0.0.1
```

**Comment by blowmage** [comment]
I don't have many thoughts on how the exported zone file should look. I'll think on it. I kinda assumed it would be grouped by record types, then sorted by domain names.

**Comment by quartzmo** [comment]
Closed by PR #322
